hatchet_sdk/worker/worker.py

```diff
@@ -196,6 +236,8 @@ async def async_start(
         if not _from_start:
             self.setup_loop(options.loop)
 
+        await self.start_health_server()
```

Comment:
do we want to always start this, or only when a flag is set or a port is set?

Reply:
good idea, I'll add a flag 👍